docs/actions.md

```diff
@@ -216,10 +216,17 @@ ok: invoked /whisk.system/samples/greeting with id 5975c24de0114ef2b5c24de0118ef
 A blocking invocation request will _wait_ for the activation result to be available. The wait period
 is the lesser of 60 seconds or the action's configured
 [time limit](reference.md#per-action-timeout-ms-default-60s).
-The result of the activation is returned if it is available within the wait period.
+
+The result of the activation is returned if it is available within the blocking wait period (this wait period is by default 60 seconds as well).
```

Comment:
does it make sense to fold this line into the sentence on 216 - a bit redundant IMO.

Reply:
AFAIK, the action default timeout is 60 seconds, the blocking wait timeout is by default 60 as well. It could certainly be worded differently but I think it's good as it is.

Reply:
just seems redundant as line 217 already says 60 seconds (you can add this is the default there) and strictly not correct since an action with a timeout less than 60s for an action, it's a little confusing as worded.